tools/tmp_cleanup

```diff
@@ -14,6 +14,9 @@ rm -rf libcosim_*
 # FMI4j (fmu-proxy)
 rm -rf fmi4j_*
```

Comment:
Is this still needed? (If so, maybe change "fmu-proxy" to "proxy-fmu" in the line above.)

Reply:
May be left behind by wrapped FMUs